cluster/kubemark/pre-existing/config-default.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Configuration for landing a Kubemark cluster on a pre-existing Kubernetes
+# cluster.
+
+# Pre-existing cluster expects a MASTER_IP
+MASTER_IP="${MASTER_IP:-}"
+MASTER_IP_AND_PORT="${MASTER_IP_AND_PORT:-$MASTER_IP:6443}"
```

Comment:
I set it to 6443 because with an assumptions of the Kubernetes cluster running.  This could be solved with the user setting ```MASTER_IP``` to include the port.  So I suppose documentation here could be an alternative.
```
clusters:                                                                                                                                                                                                          
- name: kubemark                                                                                                                                                                                                   
  cluster:                                                                                                                                                                                                         
    certificate-authority-data: "${CA_CERT_BASE64}"                                                                                                                                                                
    server: https://${MASTER_IP_AND_PORT} 
```